qa/tasks/vstart_runner.py

```diff
@@ -773,6 +768,31 @@ def list_connections():
 
         self.gather_mount_info()
 
+    def _set_fuse_daemon_pid(self):
+        # NOTE: When a command <args> is launched with sudo, two processes are
+        # launched, one with sudo in <args> and other without. Make sure we
+        # get the PID of latter one.
+        i, _break = 0, False
+        while i < 5:
+            time.sleep(5)
+
+            lines = self.client_remote.run(
+                args=['ps', '-ef', Raw('|'), 'grep', 'ceph-fuse'],
+            stdout=StringIO()).stdout.getvalue().strip().split('\n')
+            for line in lines:
+                if 'sudo' not in line and self.mountpoint in line and \
+                   'client.' + self.client_id in line:
+                    cols = line.split(' ')
+                    self.fuse_daemon.subproc.pid = int([c for c in cols if c != ''][1])
+                    _break = True
+
+            if _break:
+                break
+            i += 1
```

Comment:
@rishabh-d-dave 
I am thinking in the loop can we use the admin_socket() helper here and then parse the PID from the asok_path instead of checking the PID from the `ps -ef` ?

Will it make sense ?